src/theory/arith/rewrites

```diff
@@ -4,13 +4,13 @@
 ; interchangable. This is because Int and Real are not comparable in our
 ; type system.
 
-(define-rule arith-plus-zero ((t ? :list) (s ? :list)) (+ t 0 s) (+ t s))
```

Comment:
Is this rule not needed anymore?

Reply:
Yes, it was created prior to ARITH_POLY_NORM.  There is no need to duplicate it for Int, Real.